youtube_dl/extractor/francetv.py

```diff
@@ -80,7 +80,7 @@ def _extract_video(self, video_id, catalogue):
 
         return {
             'id': video_id,
-            'title': info['titre'],
+            'title': "%s - %s" % (info['titre'],info['sous_titre']),
```

Comment:
`sous_title` may be [empty string](http://www.francetvinfo.fr/replay-jt/france-3/soir-3/jt-grand-soir-3-lundi-26-aout-2013_393427.html).